### PR TITLE
HLSL Vector Type

### PR DESCRIPTION
**Pull request by anupamachandra**
Proposing HLSL vector types with length > 4.

### CODE REVIEW

**Review by damyanp — APPROVED** [review]
This looks ready to merge in so we can begin iterating on it.  Thanks!

File: proposals/0026-hlsl-vector-type.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<!-- {% raw %} -->
+
+* Proposal: [0026-HLSL-Vectors](0026-hlsl-vector-type.md)
+* Author(s): [Anupama Chandrasekhar](https://github.com/anupamachandra)
+* Sponsor: Damyan Pepper
```
Comment (llvm-beanz) [inline proposals/0026-hlsl-vector-type.md:5]:
```suggestion
* Sponsor: [Damyan Pepper](https://github.com/damyanp)
```

**Review by llvm-beanz — APPROVED** [review]
I also think this is good enough to merge, and start iterating on.

**Review by pow2clk — APPROVED** [review]
Thanks @anupamachandra!

**Pull request merged**